leshan-server-cluster/src/main/java/org/eclipse/leshan/server/cluster/RedisTokenHandler.java

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Sierra Wireless and others.
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ * 
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ * 
+ * Contributors:
+ *     Sierra Wireless - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.leshan.server.cluster;
+
+import java.util.Arrays;
+
+import org.eclipse.leshan.server.client.Client;
+import org.eclipse.leshan.server.client.ClientRegistryListener;
+import org.eclipse.leshan.server.client.ClientUpdate;
+import org.eclipse.leshan.util.Validate;
+
+import redis.clients.jedis.Jedis;
+import redis.clients.util.Pool;
+
+/**
+ * Only one cluster instance can be responsible of a given LWM2M client. (This restriction is mainly due to the DTLS
```

Comment:
nly one cluster instance can be responsible of a given LWM2M client. ""at a given moment"" ?